src/components/Sidebar/Sidebar.stories.tsx

```diff
@@ -35,7 +37,23 @@ const SidebarMeta = {
 } as ComponentMeta<typeof Sidebar>
 
 const Template: ComponentStory<typeof Sidebar> = () => {
-  return <Sidebar />
+  return (
+    <LoginContext.Provider
+      value={{
+        logout: async () => {
+          return undefined
+        },
+        userId: "username",
+        email: "user@open.gov.sg",
+        contactNumber: "98765432",
+        verifyLoginAndSetLocalStorage: async () => {
+          return undefined
+        },
```

Comment:
see above comment

Reply:
This is actually unused by the sidebar though - it's just required to be defined, so I've left it as is